homework/english/raby_alexandr/lection1/vocabulary.md

```diff
@@ -0,0 +1,17 @@
+1. I boot Win10 before work.
+2. I create my first homework in Github.
+3. Cannot connect to the specified server.
+4. We are debuging their code now. 
+5. She is developing new application at the moment.
+6. He disables his PC before going to bed.
+7. I am disconnecting its the network now.
```

Comment:
Seems something is wrong here, i cant understand  what u wanted to write

Reply:
Perhaps you need to write "it's"